### PR TITLE
tipocket-ctl: adapt to tipocket new feature, base64 config

### PR DESCRIPTION
**Pull request by cosven**

### CODE REVIEW

**Comment by cosven** [comment]
@ofey404

File: tipocket-ctl/tpctl/build.py [hunk 1]
```diff
@@ -32,9 +32,6 @@ def get_howto_cmds(self):
         bin_dir = self._env.dir_bin
         cmds = []
         if self._build_image:
-            for config_filepath in self._config_filepaths:
```
Comment (ofey404) [inline tipocket-ctl/tpctl/build.py:35]: We assume that user would place config file under `tipocket/config` directory by themselves, so we delete those code?

**Comment by ofey404** [comment]
Looks good to me.

**Review by ofey404 — COMMENTED** [review]
Looks good to me.
I'm not very clear about deleting the `cp configfile` commands

File: tipocket-ctl/tpctl/build.py (hunk 1 above)
Comment (cosven) [reply tipocket-ctl/tpctl/build.py:35]: Originally, we cp `tipocket/config/tpctl-xxx.toml` to `tipocket/build/config/tp-xxx.toml` to build a valid tipocket image.

**Pull request merged**